### ISSUE TITLE
[04] [Code] add the fine-tuning code to include KITTI-360 images with weak labels.

### ISSUE THREAD

**Issue by SantoshParit** [DLL-Project-Incremental-Learning/DeeplabV3_Cityscapes#4]
- Try to add the finetuning part of the code.
- Train (fine tune) using the KITTI-360 image and check if model trains.
